src/buildercore/terraform.py

```diff
@@ -401,27 +398,38 @@ def render_bigquery(context):
         } for dataset_id, options in context['bigquery'].items()
     }
 
-    if tables:
+    def add_table(table_id, table_options):
+        schema = table_options['schema']
+        stackname = context['stackname']
+        fqrn = "%s_%s" % (table_options['dataset_id'], table_id) # 'fully qualified resource name'
+
+        if schema.startswith('https://'):
+            # remote schema, add a 'http' provider and have terraform pull it down for us
+            # https://www.terraform.io/docs/providers/http/data_source.html
+            tf_file['data'][DATA_TYPE_HTTP][fqrn] = {'url': schema}
+            schema_ref = '${data.http.%s.body}' % fqrn
+
+        else:
+            # local schema. the `schema` is relative to `PROJECT_PATH`
+            schema_path = join(PROJECT_PATH, schema)
```

Comment:
given `PROJECT_PATH` is the current working directory, this `join` should not be necessary

Reply:
implicit vs explicit. joining with `PROJECT_PATH` makes an absolute path